terraform/gcp/admin_dns.tf

```diff
@@ -0,0 +1,69 @@
+resource "google_compute_global_address" "admin_ip_address" {
+  name = "admin-ip"
+}
+
+resource "google_dns_managed_zone" "juniper_admin_dns_zone" {
+  description = "juniper-admin-dns-zone"
+  dns_name    =  "${var.admin_url}."
+  dnssec_config {
+    kind          = "dns#managedZoneDnsSecConfig"
+    non_existence = "nsec3"
+    state         = "off"
```

Comment:
this is off because apparently you can't change to a new dns zone with dnssec on. it takes 24 hours to propagate - then I can switch domain to this dns zone - then in 24 MORE hours I can turn this back on. That's when the "fix dns" PR will come back online and we'll be able to go to `https://www.juniper-cmi.dev` once again.